pkg/apis/config/metrics.go

```diff
@@ -52,7 +52,7 @@ const (
 	// when it isn't specified in configmap
 	// TBD: Change to pipeline in next
 	// relase and pipelinerun level will be deprecated
-	DefaultPipelinerunLevel = "pipelinerun"
+	DefaultPipelinerunLevel = "pipeline"
```

Comment:
NIT: please drop the TBD from here - lines 53 and 54

Reply:
Resolved. :)